### PR TITLE
Update HTML/CSS

### PR DESCRIPTION
**Pull request by skoolbus39**
- Remove old and/or unused classes and styles
- Replace some custom styles with utility classes
- Replace font icons with SVGs
- Remove icon font embed in events CSS and add icon font embed to jquery-ui.min-custom.css until replaced with SVGs
- Adjust colors and spacing
- Conditionally add `<header>` element if subhead is present
- Adjust styles for event heading and subhead

### CODE REVIEW

**Comment by jsturek** [comment]
To help with cache busting, update the version in `src/UNL/UCBCN/Frontend/Controller.php`: `public static $version = '2020-01-15';`

**Comment by jsturek** [comment]
Screen Shot 2020-01-31 at 11 08 23 AM
Calendar icon not right justified on the upcoming events page (/upcoming)

File: www/templates/default/hcalendar/Upcoming.tpl.php [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <div class="section-heading">
     <h2 class="upcoming-heading">Upcoming Events</h2>
     <div class="links">
-        <a href="<?php echo $frontend->getCalendarURL(); ?>upcoming/.ics"><span class="wdn-icon-calendar" aria-hidden="true"></span><span class="dcf-sr-only">ics format for upcoming events</span></a>
+        <a href="<?php echo $frontend->getCalendarURL(); ?>upcoming/.ics" aria-label="I C S format for upcoming events">
+            <svg class="dcf-h-4 dcf-w-4 dcf-fill-current" focusable="false" width="24" height="24" viewBox="0 0 24 24"><path d="M23.5 2H20V.5a.5.5 0 00-.5-.5h-3a.5.5 0 00-.5.5V2H8V.5a.5.5 0 00-.5-.5h-3a.5.5 0 00-.5.5V2H.5a.5.5 0 00-.5.5V7h24V2.5a.5.5 0 00-.5-.5zM7 4H5V1h2v3zm12 0h-2V1h2v3zM0 23.5a.5.5 0 00.5.5h23a.5.5 0 00.5-.5V8H0v15.5zM7 15h4v-4a1 1 0 012 0v4h4a1 1 0 010 2h-4v4a1 1 0 01-2 0v-4H7a1 1 0 010-2z"></path></svg>
```
Comment (jsturek) [inline www/templates/default/hcalendar/Upcoming.tpl.php]: Missing `</a>`

File: www/templates/default/hcalendar/EventInstance.tpl.php [hunk 2]
```diff
@@ -1,44 +1,46 @@
...
-			<figure><img class="event_description_img" src="<?php echo $imageURL ?>" alt="image for event <?php echo $context->event->id; ?>" /></figure>
-		<?php endif; ?>
-        <p class="download">
-			<a class="dcf-btn dcf-btn-primary" href="<?php echo $url ?>.ics">Download this event to my calendar</a>
-		</p>
-	</div>
-</div>
+    <div class='vcalendar'>
+        <div class='vevent'>
+            <?php if (isset($context->event->subtitle)): ?><header><?php endif; ?>
```
Comment (jsturek) [inline www/templates/default/hcalendar/EventInstance.tpl.php]: Not sure if conditional is correct unless you only want the `<header>`  tag to render when event has a subtitle.

Comment (skoolbus39) [reply www/templates/default/hcalendar/EventInstance.tpl.php]: That is correct: only render the `<header>` if a subhead is present.

**Pull request merged**